Fix build failure on OS X Yosemite and earlier.
MAP_ANON is the same thing as MAP_ANONYMOUS but is defined on earlier systems.
https://bugzilla.gnome.org/show_bug.cgi?id=792074
--- src/coroutine_ucontext.c.orig	2017-02-22 05:20:10.000000000 -0600
+++ src/coroutine_ucontext.c	2017-12-30 22:47:49.000000000 -0600
@@ -63,7 +63,7 @@
     co->cc.stack_size = co->stack_size;
     co->cc.stack = mmap(0, co->stack_size,
                         PROT_READ | PROT_WRITE,
-                        MAP_PRIVATE | MAP_ANONYMOUS,
+                        MAP_PRIVATE | MAP_ANON,
                         -1, 0);
     if (co->cc.stack == MAP_FAILED)
         g_error("Failed to allocate %u bytes for coroutine stack",
